pylint/checkers/refactoring/recommendation_checker.py

```diff
@@ -121,6 +121,11 @@ def _check_use_maxsplit_arg(self, node: nodes.Call) -> None:
             and isinstance(utils.safe_infer(node.func), astroid.BoundMethod)
         ):
             return
+        inferred_expr = utils.safe_infer(node.func.expr)
+        if isinstance(inferred_expr, astroid.Instance) and any(
+            i for i in inferred_expr.nodes_of_class(nodes.ClassDef)
+        ):
```

Comment:
```suggestion
        if isinstance(inferred_expr, astroid.Instance) and any(
            inferred_expr.nodes_of_class(nodes.ClassDef)
        ):
```

Checked, and it can just be this. Even shorter 😄

Reply:
Could you als import `Instance` from `bases` instead of `astroid` directly?